Pipfile

```diff
@@ -6,8 +6,8 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-ansible = "==2.9.9"
-molecule = "==3.0.4"
+ansible = "==2.9.21"
+molecule = "==3.0.6"
```

Comment:
Why not molecule 3.0.8 instead?

Reply:
Right! I've upgraded to 3.0.8 without errors